modules/openapi-generator/src/main/resources/python/python-experimental/setup.mustache

```diff
@@ -29,13 +29,15 @@ setup(
     name=NAME,
     version=VERSION,
     description="{{appName}}",
+    author="{{infoName}}",
```

Comment:
@saigiridhar21 Can we make it default to "OpenAPI Generator community" similar to what you've done in the R client templates?

Reply:
@wing328 sure.